src/components/DialogUsers/index.tsx

```diff
@@ -0,0 +1,66 @@
+import Style from "./styled";
+import Button from "@mui/material/Button";
+import { useState } from "react";
+import axios from "axios";
+
+interface User {
```

Comment:
incearca sa creez un file pentru interfete, best practice il consider asa:

```
/src/type.d/user.ts
```
de acolo faci export la interfete, interfetele pentru acelasi obiect nu trebuie sa se repete in cod